tests/integration/acp/p2p/replicator_test.go

```diff
@@ -18,23 +18,18 @@ import (
 	testUtils "github.com/sourcenetwork/defradb/tests/integration"
 )
 
-// This test documents that we don't allow setting replicator with a collections that has a policy
-// until the following is implemented:
-// TODO-ACP: ACP <> P2P https://github.com/sourcenetwork/defradb/issues/2366
-func TestACP_P2POneToOneReplicatorWithPermissionedCollection_Error(t *testing.T) {
+func TestACP_P2POneToOneReplicatorWithPermissionedCollection_LocalACP(t *testing.T) {
```

Comment:
todo: I like the documentation on what the test does (can just add that its for local acp case)

Reply:
I dont think it is worth the eye/brain-space here. This one-liner should have been in the description, and I have a large dislike for the description prop as they are very often misleading.  

It will save us time in the long run to let the test definition define itself.